src/components/SnsItem.jsx

```diff
@@ -0,0 +1,26 @@
+import facebookIcon from '../assets/facebook.svg';
+import instagramIcon from '../assets/instagram.svg';
+import twitterIcon from '../assets/twitter.svg';
+import youtubeIcon from '../assets/youtube.svg';
+
+const ICON = {
+  facebookIcon,
+  instagramIcon,
+  twitterIcon,
+  youtubeIcon,
+};
+
+const SnsItem = ({ name, address }) => {
+  const target = /^http/.test(address) ? '_blank' : '_self';
+  const rel = target === '_blank' ? 'noopener noreferrer' : '';
+
+  return (
+    <li>
+      <a href={address} target={target} rel={rel} className="item">
+        {ICON[`${name}Icon`] ? <img src={ICON[`${name}Icon`]} alt={name} /> : `${name}`}

```

Comment:
svg 자체를 넘겨서 랜더링하는 방식을 추천드립니다
```suggestion
        <Icon/>
```